docs/writing-rules/data-flow/taint-mode.md

```diff
@@ -116,37 +265,42 @@ Remember, anything can be a sink, even the index of an array access:
 pattern-sinks:
   - patterns:
     - pattern-inside: $ARRAY[$SINK]
-    - pattern: $SINK
+    - focus-metavariable: $SINK
 ```
 
 This way we tell Semgrep that we do not want arrays to be accessed with tainted indexes.
 
-### Sanitized by side-effect
+Minimizing false positives
+--------------------------
 
-Typically a sanitizer will be some function that gets tainted data and returns untainted one. But it does not need to be that way. Sometimes data gets sanitized via side-effect, and taint mode can handle this too, for example:
+Since taint mode is intra-procedural, it does not know what other functions do. By default, Semgrep assumes that taint could always propagate through other functions.
```

Comment:
```suggestion
Since taint mode is intra-procedural, it does not recognize what other functions do. By default, Semgrep assumes that taint can always propagate through other functions.
```

Reply:
Avoid modal verbs...